website/docs/r/subnet.html.markdown

```diff
@@ -63,6 +63,7 @@ resource "nutanix_subnet" "next-iac-managed" {
 * `dhcp_server_address_port`: - (Optional) Port Number.
 * `dhcp_options`: - (Optional) Spec for defining DHCP options.
 * `dhcp_domain_search_list`: - (Optional).
+* `ip_config_pool_list_ranges`: - (Optional) Range of IPs (example: 10.0.0.9 10.0.0.19).
```

Comment:
can you please provide correct examples for ip_config_pool_list_ranges. It should be like :
 ["10.0.0.9 10.0.0.19"] .